sdc/tests/test_dataframe.py

```diff
@@ -1121,6 +1121,29 @@ def test_impl(n):
         hpat_func = sdc.jit(test_impl)
         pd.testing.assert_series_equal(hpat_func(n), test_impl(n))
 
+    def test_dataframe_min(self):
+        def test_impl(df):
+            return df.min()
+        sdc_func = sdc.jit(test_impl)
+        df = pd.DataFrame({"A": [12, 4, 5, 44, 1],
```

Comment:
Since we just use Series method for each column, I prefer to use an aggregated DF with many columns that actually differ between each other, that is, each testing some specific case - ints, floats, datatimes, etc.

Reply:
@Rubtsowa Why haven't you applied the above comment? You don't need two different tests for min, you can just use one with:
```
        df = pd.DataFrame({
                           "A": [12, 4, 5, 44, 1],
                           "B": [5.0, np.nan, 9, 2, -1],
                           "C": ['a', 'aa', 'd', 'cc', None],
                           "D": [True, True, False, True, True]
         })
```